File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.py
# 2750번 문제 : 수 정렬하기
# n개의 수가 주어지면 오름차순으로 정렬하기

n=int(input())

num=[]

for i in range(n):
    x=int(input())
    num.append(x)

for j in range(n):
    for k in range(j+1,n):
        if(num[j]>num[k]):
            temp=num[j]
            num[j]=num[k]
            num[k]=temp

for p in range(len(num)):
    print(num[p])

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.js
const fs = require('fs');
const input = require('fs')
    .readFileSync(process.platform === 'linux' ? '/dev/stdin' : 'input.txt')
    .toString()
    .trim()
    .split('\n').map(Number);
    //.map((el) => el.split(' '));

input.shift()
let temp = 0;
//let count = 0;
for(let i=input.length-1;i>0;i--){
  for(let j=0;j<i;j++){
    if(input[j] > input[j+1]){
      temp = input[j]
      input[j] = input[j+1]
      input[j+1] = temp
      //[input[j], input[j+1]] = [input[j+1], input[j]]
    }
    //count++;
  }
}
console.log(input.join('\n'))
//console.log(count);